doc/news/DM-49346.feature.rst

```diff
@@ -0,0 +1 @@
+Added a function to park the LED Projector. Also added homing to the setup_calsys and turned off LEDs in the prepare_for_flat
```

Comment:
Looks like you are adding more than one feature. Please, consider splitting them in different news fragments file.